spec/features/athletes/athlete_edit_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'edit athlete', type: :feature do
+  describe ' USER STORY #' do
+    describe ' as a user/admin when I visit edit_athlete_path' do
+      before(:each) do
+        @user = User.create!(username: "Dana Howell", email: "howelld115@gmail.com", password: "test")
+
+        @stace = @user.athletes.create!(name: "Stacey Kessler", gender: "Female", feet: 5, inches: 10, weight: 175, birthday: "1987-09-06")
```

Comment:
I think it follows convention better to have it be `@athlete` instead of `@stace` and then if you have more than one athlete to number them. (e.g. `@athlete_1`, `@athlete_2`).